features/step_definitions/setup.rb

```diff
@@ -60,3 +66,14 @@
 Given(/^a stage file named (.+)$/) do |filename|
   TestApp.write_local_stage_file(filename)
 end
+
+Given(/^I make (\d+) deployments$/) do |count|
+  step "all linked files exists in shared path"
+
+  @release_paths = (1..count.to_i).map do
+    TestApp.cap("deploy")
+    run_vagrant_command("readlink #{TestApp.current_path}")
+
+    stdout.strip
```

Comment:
I think the side-effect of `stdout` changing as a result of `run_vagrant_command` is a little unexpected. I would rather see this code be more explicit, like this:

```ruby
stdout, stderr = run_vagrant_command("readlink #{TestApp.current_path}")
stdout.strip
```